packages/code-studio/src/main/AppInit.tsx

```diff
@@ -132,9 +139,13 @@ function AppInit(props: AppInitProps) {
           const workspaceStorage = new LocalWorkspaceStorage(layoutStorage);
           const commandHistoryStorage = new PouchCommandHistoryStorage();
 
-          const loadedWorkspace = await workspaceStorage.load({
-            isConsoleAvailable: sessionWrapper !== undefined,
-          });
+          const [configs, loadedWorkspace] = await Promise.all([
+            client.getServerConfigValues(),
+            workspaceStorage.load({
+              isConsoleAvailable: sessionWrapper !== undefined,
+            }),
+          ]);
```

Comment:
Just making sure either of these erroring should lead to the app rendering an error message. Should be caught by the error boundary I think?

Reply:
Yea it'll be caught just below and error shown.